templates/editor.popover.php

```diff
@@ -35,6 +35,8 @@ class="events--input h2"
 		</fieldset>
 
 		<fieldset class="events--fieldset" ng-disabled="readOnly">
+			<textarea ng-model="properties.description.value" type="text" class="advanced--input advanced--textarea"
```

Comment:
`advanced--input` and `advanced--textarea` are for the sidebar only and I don't think they have any effect here. Take a look at the location input for the appropriate classes.